jerry-core/ecma/operations/ecma-arraybuffer-object.c

```diff
@@ -36,6 +36,39 @@
  * @{
  */
 
+/**
+ * Helper function: check if the value is Array or SharedArray buffer
+ *
+ * @return true - if value is an ArrayBuffer or SharedArrayBuffer object
+ *         false - otherwise
+ */
+extern inline bool JERRY_ATTR_ALWAYS_INLINE
+ecma_is_array_or_sharedarray_buffer (const jerry_value_t value) /**< the target value */
+{
+#if JERRY_BUILTIN_SHAREDARRAYBUFFER
+  return (ecma_is_arraybuffer (value) || ecma_is_shared_arraybuffer (value));
+#else /* !JERRY_BUILTIN_SHAREDARRAYBUFFER */
+  return ecma_is_arraybuffer (value);
+#endif /* !JERRY_BUILTIN_SHAREDARRAYBUFFER */
+} /* ecma_is_array_or_sharedarray_buffer */
+
+/**
+ * Helper function: check if the value is array or sharedarray buffer object
+ *
+ * @return true - if object is an ArrayBuffer or SharedArrayBuffer object
+ *         false - otherwise
+ */
+extern inline bool JERRY_ATTR_ALWAYS_INLINE
+ecma_is_object_array_or_sharedarray_buffer (ecma_object_t *object) /**< the target object */
```

Comment:
imho: The `is` and `object` should be switched in the name. (Also the name is really weird, however I don't know any better at the moment.